Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -363,7 +363,7 @@ - (void)setUniversalLinkUrl:(NSString *)universalLinkUrl {
 }
 
 - (NSString *)sessionParams {
-    if (_sessionParams) {
+    if (!_sessionParams) {
```

Comment:
That is what it means. Session params are set while the app is running and the Branch session is active, so the only time they'll be false is if they were never set in the initSession callback. This would be if the app crashed or something. In this case, we should read them from shared prefs. Make sense?